components/collective-page/sections/Contribute.js

```diff
@@ -164,6 +165,7 @@ class SectionContribute extends React.PureComponent {
     const hasNoContributor = !this.hasContributors(contributors);
     const canContribute = collective.isActive && !isPastEvent(collective);
     const hasCustomContribution = !collective.settings?.disableCustomContributions;
+    const hasCryptoContribution = collective.settings?.disableCryptoContributions === false;
```

Comment:
This means the default behavior is disabled, is this correct?

Reply:
@kewitz : Yes, that's the idea. They need to explicitly enable in order to display the crypto card. 🤔 🙂